### ISSUE TITLE
Textures are borked

### ISSUE THREAD

**Issue by Mixter213** [modmuss50/OptiFabric#30]
fabric-api-0.3.0+build.187
ModMenu-1.6.2-92
optifabric-1.14.3
OptiFine_1.14.3_HD_U_F1
Shaders : Internal

Textures seem to not like shaders. Only happens with the optifabric mod.
https://i.imgur.com/vayt5Ve.png

**Comment by modmuss50** [comment]
I cannot repoduce this, does it happen when shaders are set to off?
Are you using a custom texture pack?

**Comment by Mixter213** [comment]
Happens with any shaders. does not happen when shaders are disabled. hense the use of internals. no texture pack. and what i listed aobve is all im using

**Comment by lizfinite** [comment]
I'm unable to replicate this issue, either.
Used multiple levels of Sildur's Shaders, and intneral. No texture issues I found.

Tested with many many mods as well as only the mods listed, within the latest Fabric Loader release (1.14.3+build.1_yarn 0.4.8+build.155) (oops this wasn't... 7 looks very similar to 1 when just glancing okay!)

Sildur's Extreme with Volumetric Lighting:
![2019-06-28_13 04 05](https://user-images.githubusercontent.com/36245622/60361982-57035300-99a5-11e9-842c-37db58e3d6db.png)
With Debug (not that I'm sure any of the information will help):
![2019-06-28_13 04 14](https://user-images.githubusercontent.com/36245622/60362015-6aaeb980-99a5-11e9-9894-016742c4ca4d.png)

As well, internl:
![2019-06-28_13 06 49](https://user-images.githubusercontent.com/36245622/60362106-a8abdd80-99a5-11e9-8b07-c31a40411521.png)
With Debug:
![2019-06-28_13 06 59](https://user-images.githubusercontent.com/36245622/60362117-af3a5500-99a5-11e9-852c-e4f6a75e7e95.png)
